src/Nethermind/Nethermind.Consensus/Processing/BlockchainProcessor.cs

```diff
@@ -66,6 +66,7 @@ public class BlockchainProcessor : IBlockchainProcessor, IBlockProcessingQueue
         private DateTime _lastProcessedBlock;
 
         private int _currentRecoveryQueueSize;
+        private const int MaxBlocksDuringFastSyncTransition = 2048;
```

Comment:
Are there cases where there could legitimately be a case where it need to process a lot of block? Assuming 33kb per block (which is what I measured currently from DB, not sure if that is accurate), I'd err towards having a higher limit like 8000, which should take 0.25 GB, more or less, which is also slightly more than a day's worth of blocks.

Reply:
okay, make sense. I will increase it :)